### ISSUE TITLE
Display table breaks when description is too big

### ISSUE THREAD

**Issue by XeR** [TFNS/CTFNote#111]
![very long description](https://user-images.githubusercontent.com/34778827/143858556-eeb22f81-3c33-49db-b204-6946ec49d3f6.png)
If the description contains very long lines, then the table will become scrollable. All the important buttons are now out of the view...

We can solve this by wrapping the text, but I don't know if this is preferred since it may result in very high rows. Another way of solving this is too add some expansion button for the description, just like this example: https://quasar.dev/vue-components/table#expanding-rows
Or maybe cut the text at some point and when the user clicks it, a popup will be displayed with the full description?

_Originally posted by @JJ-8 in https://github.com/TFNS/CTFNote/pull/110#pullrequestreview-817671233_

**Comment by JJ-8** [comment]
I did some testing to see if I could find a decent solution and I have now the following:

![image](https://user-images.githubusercontent.com/34778827/144708143-02c00b95-a514-4416-850d-3a8462255ef0.png)

The buttons will wrap to always 2x2. This will prevent an ugly 1x3 layout and an unpractical 1x4 layout. Since this result in the row being a bit higher by default, you don't have the problem often that the rows are very uneven of height because of a variable description length.

Another screenshot when a task does not contain 5 paragraphs of lorem ipsum:

![image](https://user-images.githubusercontent.com/34778827/144708202-f1c0961f-c49f-4fb3-b2a8-a8686c9be73f.png)

I have also converted `<pre>` to `<div>` since that was required for the word wrapping for the description (#113).

Any thoughts about this? Is this better than vertical scrolling?
I'll create a PR for this if this is a good solution for this issue (and it is accepted that `<pre>` becomes `<div>`).
